src/authorize.js

```diff
@@ -37,16 +37,40 @@
       url += '&state=' + encodeURIComponent(redirectUri.substring(hashPos+1));
     }
     url += '&response_type=token';
+
+    if (global.cordova) {
```

Comment:
I think this can also be `phonegap`. Not sure if that has `cordova` then. @galfert Do you know that?